modules/drivers/mongo/src/metabase/driver/mongo.clj

```diff
@@ -237,6 +240,12 @@
   (let [version (db-major-version db)]
     (and (some? version) (>= version 5))))
 
+(defmethod driver/database-supports? [:mongo :now]
+  ;; The $$NOW aggregation expression was introduced in version 4.2.
+  [_ _ db]
+  (let [version (some-> (db-version db) parse-version)]
+    (and (some? version) (>= (first version) 4) (>= (second version) 2))))
```

Comment:
```suggestion
    (and (some? version) (nat-int? (compare version [4 2])))))
```
would work if `parse-version` used `mapv`.

Reply:
Nice, although I like how the original breaks if `parse-version` returns a one-vector. The original guarantees `version` must have major and minor parts because it raises a null pointer exception if `(second version)` is nil